config/api/src/main/java/org/jboss/windup/config/metadata/Rules.java

```diff
@@ -0,0 +1,92 @@
+package org.jboss.windup.config.metadata;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.jboss.windup.config.WindupRuleProvider;
+import org.jboss.windup.config.phase.Implicit;
+import org.jboss.windup.config.phase.RulePhase;
+
+/**
+ * A descriptor for {@link WindupRuleProvider} metadata. Can be overridden by overriding methods of {@link WindupRuleProvider}
+ * like <code>getExecuteAfter()</code>, <code>getExecuteAfterIDs()</code>, <code>getID()</code> or <code>enhanceMetadata()</code>.
+ *
+ *  @author Ondrej Zizka, ozizka at redhat.com
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Rules
+{
+    /**
+     * Returns a unique identifier for this particular rule provider.
+     * The default is based on the addon and classname,
+     * but this can be overridden here or in subclasses to provide a more readable name.
+     */
+    String id() default "";
+
+    /**
+     * Return the {@link RulePhase} in which the rules from this provider should be executed.
+     * <p>
+     * The default phase is {@link RulePhase#MIGRATION_RULES}.
+     */
+    //RulePhase phase() default WindupRuleProvider.DEFAULT_PHASE;
+    Class<? extends RulePhase> phase() default Implicit.class;
```

Comment:
The commented line should go away, and this should be set to "WindupRuleProvider#DEFAULT_PHASE". Also, the code comment needs to be updated.